src/applications/vaos/services/referral/index.js

```diff
@@ -0,0 +1,29 @@
+import { apiRequestWithUrl } from '../utils';
+
+const acheronHeader = {
```

Comment:
We don't need Acheron headers for calls to wellhive. These are only needed for calls to vistA

Reply:
Gotcha!